spec/operators/window-spec.ts

```diff
@@ -201,28 +214,6 @@ describe('Observable.prototype.window', () => {
     expectSubscriptions(closings.subscriptions).toBe(closingSubs);
   });
 
-  it('should dispose window Subjects if the outer is unsubscribed early', () => {
-    const source = hot('--a--b--c--d--e--f--g--h--|');
-    const sourceSubs = '^        !                 ';
-    const expected =   'x---------                 ';
-    const x = cold(    '--a--b--c-                 ');
-    const unsub =      '         !                 ';
-    const late =  time('---------------|           ');
-    const values = { x: x };
-
-    let window;
-    const result = source.window(Observable.never())
-      .do((w: any) => { window = w; });
-
-    expectObservable(result, unsub).toBe(expected, values);
-    expectSubscriptions(source.subscriptions).toBe(sourceSubs);
-    rxTestScheduler.schedule(() => {
-      expect(() => {
-        window.subscribe();
-      }).to.throw(Rx.ObjectUnsubscribedError);
-    }, late);
-  });
```

Comment:
@blesh now that window's internal Subjects aren't being tracked as if they were Subscriptions, their `unsubscribe` method is never called when the subscriber disposes. This test attempts to subscribe to one of the window Subjects after the outer window subscription has been disposed, assuming the `WindowSubscriber` has called `unsubscribe` on the Subject. This PR invalidates that assumption.